src/legacy/server/config/schema.js

```diff
@@ -53,9 +49,9 @@ export default () => Joi.object({
   }).default(),
 
   csp: Joi.object({
-    rules: Joi.array().items(Joi.string()).default(DEFAULT_CSP_RULES),
-    strict: Joi.boolean().default(DEFAULT_CSP_STRICT),
-    warnLegacyBrowsers: Joi.boolean().default(DEFAULT_CSP_WARN_LEGACY_BROWSERS),
+    rules: Joi.array().items(Joi.string()).default(DEFAULT_CSP_OPTIONS.rules),
+    strict: Joi.boolean().default(DEFAULT_CSP_OPTIONS.strict),
+    warnLegacyBrowsers: Joi.boolean().default(DEFAULT_CSP_OPTIONS.warnLegacyBrowsers),
```

Comment:
I believe these should be set to `HANDLED_IN_NEW_PLATFORM` similar to other options here.